lib/KeyboardAwareHOC.js

```diff
@@ -504,11 +504,17 @@ function KeyboardAwareHOC(
         })
       }
       const refProps = { [hocOptions.refPropName]: this._handleRef }
+
+      //support inverted views
+      const contentInset = this.props.inverted ?
+      { bottom: this.state.keyboardSpace } :
```

Comment:
I guess this have to be vice-versa like below!

const contentInset = this.props.inverted ?
      { top: this.state.keyboardSpace } : 
      { bottom: this.state.keyboardSpace }